addons/hr_work_entry_contract/static/src/js/work_entries_controller_mixin.js

```diff
@@ -88,11 +88,18 @@ odoo.define('hr_work_entry_contract.WorkEntryControllerMixin', function(require)
         },
 
         _regenerateWorkEntries: function () {
+            let only_employee = null;
+            let viewed_employees = Object.keys(this.model.allRows)
+            if (viewed_employees.length === 1) {
+                only_employee = JSON.parse(viewed_employees[0])[0].employee_id[0]
+            }
```

Comment:
Only a suggestion:
It's not really clear at first glance what is being extracted, I think it's a bit clearer this way :innocent: 

```suggestion
            const all_rows = Object.values(this.model.allRows);
            const only_employee = all_rows.length === 1 ? viewed_employees[0].resId : null;
```